repos/system_upgrade/el7toel8/actors/repositoriesblacklist/libraries/library.py

```diff
@@ -13,6 +13,11 @@ def _get_list_of_optional_repos():
     repo_map = next(api.consume(RepositoriesMap), None)
     if repo_map:
         for repo in repo_map.repositories:
+            # FIXME: that's not enough when src system is beta/htb as in such
```

Comment:
Add conditions based on the SRC/DST prod type env vars to alter the search string.